app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt

```diff
@@ -119,9 +152,14 @@ class OrdersUnderUserFragment : Fragment(), ScrollToTop {
     }
 
     private fun showNoTicketsScreen(show: Boolean) {
-        noTicketsScreen.visibility = if (show) View.VISIBLE else View.GONE
-        findMyTickets.setOnClickListener {
-            Utils.openUrl(requireContext(), resources.getString(R.string.ticket_issues_url))
+        noTicketsScreen.isVisible = show
+        if (safeArgs.showExpired) {
+            findMyTickets.isVisible = false
+            noTicketMessage.text = getString(R.string.no_past_tickets)
```

Comment:
Do not set text on the same text view. Because `Find my tickets` is redirect to a link but `No past tickets` should not redirect. Use two text views in the same position and handle visibility accordingly.

Reply:
no past tickets is not redirecting. There are 2 text view 1 for redirecting, 1 for message. I make redirecting text view GONE and change message.

Reply:
Okay, you just user 1st text view to display the message. :+1:

Reply:
yes that's what I used.